Match.java

```diff
@@ -77,13 +77,66 @@ public void setGoalScoredAwayTeam(int goalScoredAwayTeam){
 	
 	public void infoAboutOneGame(){
 		System.out.println(season + ", "  + round + ", " + nameOfGroup + ", " + homeTeam + " - " + awayTeam + " " + goalScoredHomeTeam + ":" + goalScoredAwayTeam);
-		
+	}
+	
+	@Override
+	public int hashCode() {
```

Comment:
Not sure about this implementation as In C# I use different. But can be Java specific

Reply:
Messy, comments 17 days old. Please create new PRs otherwise difficult to review